python/sglang/srt/lora/utils.py

```diff
@@ -109,8 +113,13 @@ def get_normalized_target_modules(
 
     result = set()
     for name in target_modules:
-        normalized_name = params_mapping.get(name, name)
-        result.add(normalized_name)
+        base_name = name.split(".")[-1]
+
+        if base_name in params_mapping:
+            normalized_name = params_mapping[base_name]
+            result.add(normalized_name)
+        else:
```

Comment:
why not just use the `dict.get` method as in the original code? The overheads is trivial here, but using get reduces dict lookup twice and is also arguably more readable.